packages/core/scripts/local/EncodeParams.js

```diff
@@ -49,8 +49,8 @@ const encodeParams = async callback => {
       timerAddress: await expiringMultiPartyCreator.timerAddress()
     };
 
-    // Inject constructor params neccessary for "latest" version of the EMPCreator:
-    if (abiVersion === "latest") {
+    // Inject constructor params neccessary for "2.0.1" version of the EMPCreator:
```

Comment:
here (and in all the other bots) we can bound to this version to get anything EMP v2 or perp related.